src/NSubstitute/Core/SubstituteFactory.cs

```diff
@@ -1,4 +1,6 @@
+
```

Comment:
**question (nitpick):** several of these changed files have a blank line to added to start. Is this an editorconfig thing or accidental?